matdeeplearn/tasks/task.py

```diff
@@ -14,9 +13,11 @@ def __init__(self, config):
 
     def setup(self, trainer):
         self.trainer = trainer
-        checkpoint = self.config.get("checkpoint", None)
-        if checkpoint is not None:
-            self.trainer.load_checkpoint(self.config["checkpoint"])
+        use_checkpoint = self.config["model"]["load_model"]
```

Comment:
Let's make this optional too, so that it's only needed if a model wants to be loaded
```suggestion
        use_checkpoint = self.config["model"].get("load_model", False)
```